src/OrleansTestingHost/Extensions/TestConfigurationExtensions.cs

```diff
@@ -14,37 +14,40 @@ public static class TestConfigurationExtensions
         /// <summary>
         /// This call tweaks the cluster config with settings specific to a test run.
         /// </summary>
-        public static void AdjustForTestEnvironment(this ClusterConfiguration clusterConfig)
+        public static void AdjustForTestEnvironment(this ClusterConfiguration clusterConfig, string dataConnectionStringFallback)
```

Comment:
not sure having this override is good approach.  Following this pattern.  Opens the door to other overrides, which over time could make this code very hard to maintain.  Suggest making caller is responsible for setting the connection string in the cluster config.